mesheryctl/pkg/utils/error.go

```diff
@@ -485,3 +487,19 @@ func ErrRequestResponse(err error) error {
 		[]string{"Error occurred while generating a response"},
 		[]string{"Check your network connection and the status of Meshery Server via `mesheryctl system status`."})
 }
+
+func ErrMarshalStructToCSV(err error) error {
+	return errors.New(ErrMarshalStructToCSVCode, errors.Alert,
+		[]string{"Failed to marshal struct to csv"},
+		[]string{err.Error()},
+		[]string{"Error occured converting struct to [][]string", "The struct might have in-appropriate csv tags"},
+		[]string{"Verify the struct is non-empty", "Verify the struct has appropriate csv tags"})
+}
+
+func ErrAppendToSheet(err error, id string) error {
+	return errors.New(ErrAppendToSheetCode, errors.Alert,
+		[]string{fmt.Sprintf("Failed to append data into sheet %s", id)},
+		[]string{err.Error()},
+		[]string{"Error occured while appending to the spreadsheet", "The credentials might be incorrect/expired"},
+		[]string{"Ensure correct append range (A1 notation) is used", "Ensure correct credentials are used"})
```

Comment:
How many credentials are they using to access the spreadsheet? If just one, then rephrase.